doc/build-unix.md

```diff
@@ -254,7 +254,7 @@ Hardening enables the following features:
     randomly located as well.
 
     On an AMD64 processor where a library was not compiled with -fPIC, this will cause an error
-    such as: "relocation R_X86_64_32 against `......' can not be used when making a shared object;"
+    such as: "relocation R_X86_64_32 against `......' cannot be used when making a shared object;"
```

Comment:
This string looks being copy-pasted from other source. Should we alter it?

Reply:
Not sure, I thought about this, but the "such as:" preceding seems to make it only an example.

Reply:
Yes, it looks to me it is a compiler message. Better for search ability to leave it as is.

Reply:
Thanks, done.